vsts.yml

```diff
@@ -3,6 +3,12 @@ jobs:
   displayName: Build Electron via GN
   timeoutInMinutes: 120
   steps:
+
+  - task: CopyFiles@2
+    displayName: 'Copy Files to: src/electron'
+    inputs:
+      TargetFolder: src/electron
```

Comment:
... what are the inputs and what are the outputs of this copy operation? it looks like it's only specifying the target folder, not the source :S

Reply:
Its *magic* 🎉 .  Actually its just defaults in play, documented here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/copy-files?view=vsts